docs/updating/4_update_3.2.md

```diff
@@ -30,6 +34,29 @@ If you are updating from a version prior to 3.0, you have to implement all the c
 
     `psql <database_name> < upgrade/db/postgresql/ezplatform-3.1.0-to-3.2.0.sql`
 
+## Update entity managers
+
+Version v3.2.6 introduces new entity managers.
+To ensure that they work in multi-repository setups, you must update the GraphQL schema.
+You do this manually by following this procedure:
+
+1. Update your project to v3.2.6 and check if the container has been built. If not, run the `php bin/console cache:clear` command to generate the container.
```

Comment:
Can we quickly explain how you "check if the container has been built", or can we assume this is clear for the audience?